src/util.ts

```diff
@@ -1437,9 +1438,7 @@ export function isValidRGB(rgb: string): boolean {
  * @returns True if the string is valid, false otherwise.
  */
 export function isValidRGBA(rgba: string): boolean {
-  rgba = rgba.replace(' ', '')
```

Comment:
You removed this kind of "trim" function.

Reply:
This would allow values like ` r g b a ( 1 2 3 , 2 5 5 , 1 0 0 , 0 . 7 5 ) `. Which is definitely not valid.

Reply:
`    rgba( 12 , 255 , 100 , .75 )   ` would be valid!

You are right but you are changing functionallity here :wink: 
You should decide if you want to test existing functionality in a PR or change it :smile: 

Maybe we just keep the replace for now!?

Reply:
I change the functionality but not the contract. The function now does what it always claimed to do.

Reply:
Also I ditched the replace in RGB already. If passing invalid values due to spaces is desired we should discuss that too.

Reply:
I changed the name or this PR.

Reply:
> Also I ditched the replace in RGB already.

:scream: 

We must be careful! When we change functionality we must at least increase the major version and test if visjs modules are affected. We are working on a big library here and don't want to break things!

`isValidRGB` and `isValidRGBA` are often used in all modules to validate user input. We can not just change it without communicating it!

I would prefer fixing the regexp for both functions back so that they support whitespaces. This should be done in a separate pull-request.

Reply:
I don't know, I would consider the old behavior a bug. Plus it's not consistent with the name of the function or the docs.

Reply:
Let's discuss this in an issue and do just the tests for now. Ok? 
I would prefer it if the functionality would be not touched for now and only existing functionally would be testes. But I'm willing to merge this, but it should not be released before we discussed this further.

Reply:
Another option is to set up a new branch to prepare and test these changes and release it as a major version later.

Reply:
Hmm, we don't really validate colors using these. The only place where we use them (isValidRGB and isValidRGBA) is the color picker.

Reply:
I saw 100+ occurrences of these functions and just guessed the are used for input validation :man_facepalming: 
I think you are right. No harm here! Lets merge this!